flask/tests/test_genes.py

```diff
@@ -33,7 +33,6 @@ def test_fetch_genes_csv(test_database, client, login_as):
     login_as("user")
     response = client.get("/api/summary/genes", headers={"Accept": "text/csv"})
     assert response.status_code == 200
-    assert response.headers["Content-Type"] == "text/csv"
```

Comment:
What happened here?

Reply:
Ha good question. I think I was planning to test the`csv_response` function independently but then realized it would just be a test of flask code but forgot to restore this assertion. Restored in [61302ea](https://github.com/ccmbioinfo/stager/pull/567/commits/61302ead558ef578a390c4c18c058dea6f71a2c0#diff-93c465b7fd38919afba78a72d6b399c8e754f8dec3afe5b4b56d51b0c0969a70R36)